openwisp_radius/migrations/0023_add_identity_verification.py

```diff
@@ -0,0 +1,68 @@
+# Generated by Django 3.1.7 on 2021-02-26 05:29
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('openwisp_radius', '0022_organizationradiussettings_registration_enabled'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='organizationradiussettings',
+            name='needs_identity_verification',
+            field=models.BooleanField(
+                blank=True,
+                default=False,
```

Comment:
the default should be `None` otherwise it will always default to False even though the global setting is changed.

Reply:
Done (https://github.com/openwisp/openwisp-radius/pull/237/commits/ef3686e4dbe8c5e85a5bec1769da36eaafd87ea8)